### ISSUE TITLE
Template #1 (default): Adjusting paddings &/or margins

### ISSUE THREAD

**Issue by joeflack4** [PMA-2020/ppp#3]
## Description
Make formatting tighter with less spacing between content, again to save space.

## Tasks
- [x] 0. Develop an updated template.
- [x] 1. You have a separate branch. A branch where the template has been updated to the "all_at_once" version. Or, a branch where it doesn't replace the original ppp template, but exists as a second, template--either as the new default, or non-default.
- [x] 2. You make a pull request from your branch to the develop branch.
- [x] 3. I will review, resolve any merge conflicts, merge into develop, and close this issue.
- [x] 4. Then you can delete your feature branch if you want: `git branch -D [FEATURE_BRANCH]`, `git push origin --delete [FEATURE_BRANCH]`
- [x] 5. You will `git fetch --all` and `git pull trunk/develop` from inside your develop branch so that your develop branch is in sync with trunk.
- [x] 6. If you have any other any other currently active feature branches, you can update them to be in sync with this new feature branch by switching to those branches, and doing `git rebase develop` or `git rebase trunk/develop`. You may experience some merge conflicts that need resolution. If so, it will be a good learning experience.

**Comment by tulvit** [comment]
No good news :'(

I'll assume that "tighter formatting" == "saving space" == "less pages". Because after some changes it may seem that it's become really "tighter". But if the number of pages remains the same - no, it isn't

Original .xlsx (the one I have for tests) -> .html -> .doc == 34 pages. So it's a starting point.

### Reducing font size a bit
1em -> 0.9em
Doesn't work. Same 34 pages.

### Word breaking on '_' and other chars
In some cases long strings ('${husband_cohabit_start_recent}') produce extra space in a nearby cells, so it was worth to try...
Doesn't work. Same 34 pages.

### Changing table proportions
Original columns' ratio was 50%/50%. But question cells contain a way more content then response cell (just forms), so why not to change it to 70%/30%? I really hoped it would work, and yeah, it seems it really become a little bit tighter (and it was quite a tricky task, for example if a colspan attribute is presented, then all tds' width will be ignored) , but...
Doesn't work. Same 34 page.

### Misc.
Changing this and that, here and there...
Doesn't work, doesn't work.

### Margins/paddings
Let's go wild:
```
* {
  margin: 0 !important;
  padding: 0 !important;
}
```
Doesn't work. But, perhaps, Word just ignore this wildcard. Tested each and every tag separately - and it worked. Hooray! So,
```
td,
p,
h1,
h2,
h3,
ul,
li,
tr,
em,
et cetera {
  padding: 0 !important;
  margin: 0 !important;
}
```
and... 27 pages! 20% less!

But I'm not sure it's the case. The main problem is an overall layout.

For example, it *might* make sense to use not this:
_question|response
question|response_
table layout, but 
**question**
_response_
**question**
_response_

And yet again, there are these long vertical lists,
_[ ] male
[ ] female_

I really want to make them float, it will save so much space...
But I really don't want to make them in one line, it will reduce readability so much...
I mean this one is easy to use/submit:
_[ ] January 
[ ] February 
[ ] March 
[ ] etc._
and this is just a mess:
_[ ] January [ ] February [ ] March [ ] etc._

So I guess readability should be always over the space saving...

In other words, what should I do next?

I guess:
- Playing around with padding/margins only, anyway reducing page count by 10-15% will be not a bad result after all with options we have...
- Adding to a todo list "think hard about further layout improvements".

**Comment by joeflack4** [comment]
Yeah, I think you're right. I don't think there's too much we can do here. Thinking back to the request that was made of us, I think it will be best if we just mess with the padding and font sizes a little bit, but not a lot.

70/30 may be a good idea though, as an alternative to 50/50. But I don't think we should change much more than that.

I think with the padding of 0, it just looks worse. But before this adjustment, there was barely any padding at all.

**Can you do the following:**
- [ ] Produce a 70/30 .doc file example, but with the original padding.

After that, I'll share the following with my co-workers and get some feedback: (a) the original (50/50, default padding), (b) 70/30, 0 padding, and (c) 70/30, default padding.

**Comment by tulvit** [comment]
Files in the zip:

**original.doc** - original doc with no modifications

**original2.doc** - actually, original is a little bit "buggy", so columns' ratio is not 50%/50%, but more like 48%/52%, because some content in a response cells is too long and occupy more than 50% of the document (like three inputs in a row), in this version it's fixed

**font_reduced.doc** - same as original2, but with font size reduced by 10%

**padding_reduced.doc** - same as original2, but with padding reduced

**60_40_ratio.doc** - same as original2, but with 60/40 table ratio

**70_30_ratio.doc** - same as original2, but with 70/30 table ratio

**all_at_once.doc** - all changes at once

[docs.zip](https://github.com/PMA-2020/ppp/files/2142603/docs.zip)

**Comment by joeflack4** [comment]
@tulvit Hey Vitaly~.

IMO, I like the "all_at_once" version, which I think is--60/40, font reduced, without padding reduction.

The way I'd like to proceed with this ideally is:
- You have a separate branch. A branch where the template has been updated to the "all_at_once" version. Or, a branch where it doesn't replace the original ppp template, but exists as a second, template--either as the new default, or non-default.
- You make a pull request from your branch to the develop branch.
- I will review, resolve any merge conflicts, and merge into develop.
- You will `git fetch --all` and `git pull trunk/develop` from inside your develop branch so that your develop branch is in sync with trunk.
- If you have any other any other currently active feature branches, you can update them to be in sync with this new feature branch by switching to those branches, and doing `git rebase develop` or `git rebase trunk/develop`. You may experience some merge conflicts that need resolution. If so, it will be a good learning experience.

**Comment by joeflack4** [comment]
@tulvit Closing this issue, as it and the PR are complete. GJ!
